Data/DataLoader_mask2.py

```diff
@@ -0,0 +1,63 @@
+from pyparsing import Optional
+import pytorch_lightning as pl
+from pytorch_lightning.utilities.types import TRAIN_DATALOADERS
+from torch.utils.data import DataLoader
+from pytorch_lightning.utilities.types import EVAL_DATALOADERS, STEP_OUTPUT, TRAIN_DATALOADERS, OptimizerLRScheduler
+import pandas as pd
+import numpy as np
+from Dataset import RoadMarkingDataset
+
+class RoadMarkingDataloader(pl.LightningDataModule):
+    '''
+    Dataloader class for the Mask2Former model
+
+    '''
+    def __init__(self, train_csv:str, val_csv:str, test_csv:str,batch_size:int , image_column:str, mask_column:str):
+        '''
+        Constructor for the class
+        '''
+        super().__init__()
+        self.train_csv = train_csv
+        self.val_csv = val_csv
+        self.test_csv = test_csv
+        self.batch_size = batch_size
+        self.image_column = image_column
+        self.mask_column = mask_column
+        self.train_ds = None
+        self.val_ds = None
+        self.test_ds = None
+        # TO DO : Add the transforms (NORMALIZATION, RESIZE, TO TENSOR)
+        self.__transform = None 
+
+    def setup(self, stage: str):
+        '''
+        Setup the dataset for the dataloader
+        '''
+        if stage == 'fit':
```

Comment:
this won't use the validation dataset at all, according to the documentation at stage == 'fit' you should load both training and validation dataset in order to work properly

Reply:
then you would have a separate if for stage == 'test' like you already did